src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,366 @@
+import {
+  VaAlert,
+  VaCheckboxGroup,
+  VaModal,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { scrollAndFocus } from 'platform/utilities/ui';
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  conditionsDescription,
+  conditionsPageTitle,
+  conditionsQuestion,
+  makeTEConditionsUISchema,
+} from '../../content/toxicExposure';
+import {
+  deletedToxicExposureAlertConfirmationContent,
+  deleteToxicExposureModalContent,
+  deleteToxicExposureModalDescription,
+  deleteToxicExposureModalTitle,
+} from '../../content/toxicExposureChoiceContent';
+import { formTitle, showToxicExposureDestructionModal } from '../../utils';
+
+export const toxicExposureChoicePageTitle =
+  'Option to claim conditions related to toxic exposure';
+
+/**
+ * List of all toxic exposure related keys that need to be deleted
+ * when user opts out of toxic exposure claims.
+ * @constant {string[]}
+ */
+export const toxicExposureKeys = [
+  'conditions',
+  'gulfWar1990',
+  'gulfWar1990Details',
+  'gulfWar2001',
+  'gulfWar2001Details',
+  'herbicide',
+  'herbicideDetails',
+  'herbicideOtherLocations',
+  'additionalExposures',
+  'additionalExposuresDetails',
+  'specifyOtherExposures',
+];
+
+const ToxicExposureChoicePage = ({
+  goBack,
+  goForward,
+  data,
+  setFormData,
+  contentBeforeButtons,
+  contentAfterButtons,
+  onReviewPage,
+  updatePage,
+}) => {
+  const showDestructiveModal = useSelector(showToxicExposureDestructionModal);
+
+  const [
+    showDeleteToxicExposureModal,
+    setShowDeleteToxicExposureModal,
+  ] = useState(false);
+  const [
+    showDeletedToxicExposureConfirmation,
+    setShowDeletedToxicExposureConfirmation,
+  ] = useState(false);
+
+  const deletedToxicExposureConfirmationRef = useRef(null);
+
+  useEffect(
+    () => {
+      if (
+        showDeletedToxicExposureConfirmation &&
+        deletedToxicExposureConfirmationRef.current
+      ) {
+        deletedToxicExposureConfirmationRef.current.focus();
+      }
+    },
+    [showDeletedToxicExposureConfirmation],
+  );
+
+  const modalRef = useRef(null);
+  useEffect(
+    () => {
+      if (showDeleteToxicExposureModal && modalRef.current) {
+        // Use the ref directly instead of querying the document
+        scrollAndFocus(modalRef.current);
+      }
+    },
+    [showDeleteToxicExposureModal],
+  );
+
+  const hasToxicExposureData = () => {
+    return toxicExposureKeys.some(key => {
+      if (key === 'conditions') return false; // Skip conditions as it's the trigger
+      if (!(key in (data?.toxicExposure || {}))) return false;
+
+      const value = data.toxicExposure[key];
```

Comment:
I think all that would need to be evaluated is whether `value` is a non-empty Object.